sum/sum.py

```diff
@@ -0,0 +1,28 @@
+# shortest = {}
+# longest = {}
+
+
+# for i in range(len(words)):
+#     for j in range(len(words)):
+#         if len(words[i]) < len(words[j]):
+#             shortest[words[i]] =  shortest.get(words[i], 0) + 1
+#         elif len(words[i]) > len(words[j]):
+#             longest[words[i]] = longest.get(words[i], 0) + 1
+
+
+words = ["house", "me", "button", "mat"]
+lengths = [len(word) for word in words]
+
+
+new_tup = ()
+
+for i, length in enumerate(lengths):
```

Comment:
So using a `for` loop works here, but I would suggest simply getting the values you need independently, and then returning them in a tuple at the end. Something like:

```
max_length = ...

min_length = ...

average_length = ...

return (min_length, max_length, average_length)
```

Reply:
Hint: for these, you can do one-liners for find the values, using some of the built-in functions:

https://docs.python.org/3/library/functions.html

with composition of another function...and remember composition is this pattern:

```
nums = [1, 2, 3]

sum_of_doubles = sum(map(lambda x: x * 2, nums)
```